static/app/utils/discover/fields.tsx

```diff
@@ -74,14 +75,17 @@ export type QueryFieldValue =
   | {
       field: string;
```

Comment:
We could have a base type for these because they look almost the same.
```
type BaseField = {
 field: string,
 kind: "equation" | "field",
 alias?: string
}
```

Reply:
when updating this, I have to fix a couple of issues in other places and would prefer to don't do it in this PR